include/VerificationValidation.h

```diff
@@ -31,6 +31,7 @@ namespace VerificationValidation {
         };
 
         QString terminalOutput;
+	QString cmd;
```

Comment:
as far as I see, this shouldn't be necessary
- no references in code other than setting